sdk/src/types/block/error.rs

```diff
@@ -90,11 +82,6 @@ pub enum Error {
     InvalidUnlockConditionCount(<UnlockConditionCount as TryFrom<usize>>::Error),
     InvalidUnlockConditionKind(u8),
     InvalidFoundryZeroSerialNumber,
-    MigratedFundsNotSorted,
-    MilestoneInvalidSignatureCount(<SignatureCount as TryFrom<usize>>::Error),
-    MilestonePublicKeysSignaturesCountMismatch { key_count: usize, sig_count: usize },
-    MilestoneOptionsNotUniqueSorted,
-    MilestoneSignaturesNotUniqueSorted,
     MissingAddressUnlockCondition,
     MissingGovernorUnlockCondition,
     MissingPayload,
```

Comment:
Can't comment there, but `TailTransactionHashNotUnique` is also only required for milestones

Reply:
Done